frontend/catalyst/jax_tracer.py

```diff
@@ -400,16 +400,19 @@ def lower_jaxpr_to_mlir(jaxpr, func_name):
     return mlir_module, ctx
 
 
+# pylint: disable=too-many-arguments
 @debug_logger
-def trace_quantum_tape(
+def trace_quantum_operations(
     quantum_tape: QuantumTape,
     device: QubitDevice,
     qreg: DynamicJaxprTracer,
     ctx: JaxTracingContext,
     trace: DynamicJaxprTrace,
+    qnode: qml.QNode = None,
```

Comment:
Should the qnode ever be none?

Reply:
Not as far as I know. I will make it a positional argument.